contentcuration/contentcuration/frontend/channelEdit/components/AddRelatedResourcesModal.vue

```diff
@@ -147,7 +147,8 @@
         channel_id: this.currentChannelId,
       });
 
-      this.selectedNodeId = this.getContentNodeParents(this.targetNodeId)[0].id;
+      const ancestors = this.getContentNodeAncestors(this.targetNodeId);
+      this.selectedNodeId = ancestors[ancestors.length - 2].id;
```

Comment:
I'm not sure I understand the meaning of this. The original just looks like it's getting he parent id? Which if that's the case, this can be simplified.

Reply:
The reasons for this update are:
1. `getContentNodeParents` and `getContentNodeAncestors` return nodes in the opposite sort order - that's why there's a need to reach to `[ancestors.length - 1]` instead of `[0]`
2. `getContenNodeAncestors` doesn't return only parent nodes, it actually returns also a node for which I want to receive its ancestors so there is one extra node - that means that I actually need to use `[ancestors.length -2]` instead of `[ancestors.length-1]`

Reply:
@MisRob So for point (2), doesn't that mean you're simply getting the parent node of `this.targetNodeId`?

Reply:
I'm going to go ahead and merge this. I was just curious if it could be simplified to `this.selectedNodeId = getTreeNode(this.targetNodeId).parent`?

Reply:
Oh yes, that looks definitely more readable.